lib/recurly/dunning_campaign.rb

```diff
@@ -0,0 +1,33 @@
+module Recurly
+  # Recurly Documentation: https://developers.recurly.com/api-v2/latest/index.html#tag/dunning-campaigns
+  class DunningCampaign < Resource
+    # @return [[DunningCycle], []]
+    has_many :dunning_cycles
+
+    # @return [[Plan], []]
+    has_many :plans
+
+    define_attribute_methods %w(
+      id
+      name
+      code
+      description
+      default
+      dunning_cycles
+      created_at
+      updated_at
+      deleted_at
+      plans
+    )
+
+    def bulk_update(plan_codes)
+      return false unless link? :bulk_update
```

Comment:
can you explain what this is doing?

Reply:
It is checking if there is an action link for `:bulk_update` as part of the response when instantiating the resource.